docs/UserGuide.md

```diff
@@ -6,7 +6,7 @@
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-2. Download the latest `BankWithUs.jar` from [here](https://github.com/AY2223S2-CS2113-T13-3/tp/releases).
+2. Download the latest `tp.jar` from [here](https://github.com/AY2223S2-CS2113-T13-3/tp/releases).
```

Comment:
I think might be better to leave it as BankWithUs.jar in the UG and instead change the name of the jar file to BankWithUs before release.

Reply:
Ok sure!

Reply:
I changed it back to BankWithUs.jar just now.